### DISCUSSION TITLE
How can I contribute to this project?

### DISCUSSION THREAD

**Discussion by pokhreli** [kiranshahi/Employee-Management-System#22] [Q&A] [answered]
I am interested in contributing this project, can you guide me how can I contribute?

**Comment by kiranshahi** [comment, marked as answer]
Hi, thank you for your interest in this project.

In order to contribute, you need to [fork](https://github.com/kiranshahi/Employee-Management-System/fork) this project to your account and clone it to your local system.

For more details please go through [contribution guide](https://github.com/kiranshahi/Employee-Management-System/blob/master/CONTRIBUTING.md).
